web/app/themes/ppj/functions.php

```diff
@@ -6,7 +6,14 @@ function enqueue_scripts()
     $legName  = getLegNameFromPath();
     $root_dir = get_template_directory_uri() . '/dest';
     wp_enqueue_style( 'main-css', $root_dir . mix_asset( "/css/{$legName}.css" ), null, null );
+
+    wp_enqueue_script( 'header-js', $root_dir . '/js/header.js', null, null, false );
     wp_enqueue_script( 'main-js', $root_dir . mix_asset( '/js/main.js' ), null, null, true );
+    wp_enqueue_script( 'wistia', '//fast.wistia.com/assets/external/E-v1.js', null, null, true );
+
+    if (is_page('Find a job')) {
```

Comment:
Change this to use `is_page_template('find-a-job.php')` to couple it with the page template instead of page title. This will avoid it breaking if the page title is changed.

Reply:
Done